core/src/main/java/org/elasticsearch/bootstrap/BootstrapCheck.java

```diff
@@ -470,16 +471,44 @@ public String errorMessage() {
 
     }
 
-    abstract static class MightForkCheck implements BootstrapCheck.Check {
+    abstract static class IsSeccompInstalledCheck implements BootstrapCheck.Check {
+
+        // visible for testing
+        boolean isSeccompInstalled() {
```

Comment:
Definitely not, I started with this check and the might fork checks sharing more code but decided that I didn't like it and this is a remnant of that effort. I will remove.